### DISCUSSION TITLE
wire:ignore not-working

### DISCUSSION THREAD

**Discussion by bahramiholding** [livewire/livewire#3293] [7. Bugs (archived)]
Hi everyone
I used livewire in one of my projects, but by putting wire: ignore, the data related to the component is still loaded. And even putting wire: key did not solve this problem. Especially for working with text editor - (‌ ckeditor) or other editors

         <div class="col-12" >
            <div class="form-group"
                 wire:ignore
                 wire:key="Body"
            >
                <label>Body  :</label>
                <textarea
                    id="body"
                    class="form-control"
                    name="body" rows="3"
                    wire:model.defer="body"
                ></textarea>
                @error('body')<span class="text-error d-flex">{{ $message }}</span> @enderror
            </div>
        </div>

livewire version = 2.5
laravel version  = 8.49.2

**Comment by joshhanley** [comment]
@a-bahrami I'm not sure I understand, what data are you talking about that is still loaded? Any change you can share an example of what you mean and a description of what you would hope to happen using https://laravelplayground.com/

Thanks!
